# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/xmldecl/.cargo-checksum.json
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 10
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/third_party/rust/xmldecl/.cargo-checksum.json b/third_party/rust/xmldecl/.cargo-checksum.json
--- a/third_party/rust/xmldecl/.cargo-checksum.json
+++ b/third_party/rust/xmldecl/.cargo-checksum.json
@@ -1,1 +1,1 @@
-{"files":{"COPYRIGHT":"12d3af8e034aee1d1955deb3797cb32fd7e014d1540a8714971238a35201fa0f","Cargo.toml":"95e515226b050b3c5e9bac1f8746663e31d88cdf89b304bf9344bfb9b7a71d9f","LICENSE-APACHE":"cfc7749b96f63bd31c3c42b5c471bf756814053e847c10f3eb003417bc523d30","LICENSE-MIT":"3fa4ca83dcc9237839b1bdeb2e6d16bdfb5ec0c5ce42b24694d8bbf0dcbef72c","README.md":"2b796024403be403370734e4e7f2ed9d668509ad256c903c80b131782ef8d241","src/lib.rs":"61b0d89890c57687ad04a27db31c9ee55e39ec0156c63db77a3176887dfacb3d"},"package":"e8c855a4ffa155714197d09677f21262f1d2e8d553fe5356c1752ad61f8a868c"}
\ No newline at end of file
+{"files":{"COPYRIGHT":"12d3af8e034aee1d1955deb3797cb32fd7e014d1540a8714971238a35201fa0f","Cargo.toml":"0bb4ff44435bd07eda06de25599dc80a3954b7a55ab317a7276d9c2b239f49a5","LICENSE-APACHE":"cfc7749b96f63bd31c3c42b5c471bf756814053e847c10f3eb003417bc523d30","LICENSE-MIT":"3fa4ca83dcc9237839b1bdeb2e6d16bdfb5ec0c5ce42b24694d8bbf0dcbef72c","README.md":"039f7f51a3e74690d16fbbc35edf42ff7ee23024aeff60a1320fc19117c69e8a","src/lib.rs":"da20ff71c6d81c6a7f066d15ded76bdb8d447784a843a10fc78ca50c7a4f4015"},"package":"efeb408acbc94f7459f1a3ee3620c108ebea5e5baf93a4641c07d57f59f5ffd1"}
\ No newline at end of file